_posts/blog/2021-01-18-Open-position-Cryptography-Engineer.md

```diff
@@ -0,0 +1,89 @@
+* Strong testing skills, both in writing automated tests and using techniques such as fuzzing to catch bugs. 
+* Team player.
+
+### Nice-to-haves
+
+* Rust programming experience.
+* Experience designing and/or implementing complex distributed systems.
+* Experience designing and/or implementing complex zero-knowledge proof systems.
+* Deep technical understanding of internet privacy tools, with a particular focus on finance/cryptocurrency.
+* Strong written and oral communication skills.
+
+### Interview process
+
+We depend on work samples and collaborative interviews to evaluate the candidate. 
+[Please submit the following to this form](https://docs.google.com/forms/d/e/1FAIpQLSelpDkmqjgVgiTfVFukB9TbIoIExWxVDHn0VvnSboO4nJIN1A/viewform).
+* Software sample
+* Writing sample
+* Analysis sample
+* Presentation sample
+* Resume (or equivalent)
+
+We will follow up within a week of your submission.
+##### Work Samples
+We are asking to see samples of work that you have already done. Feel free to send more than one sample of each if that best reflects your accomplishments relevant to the position.
+Here’s what we want to see in detail:
+
+*Software Sample*
+
+We expect this position to contribute to producing engineering artifacts that meet the needs of our projects. We use past work to evaluate how you will likely perform in future work for the foundation. If you don't have existing public work, that's OK -- just pick something you've done and describe it for us in as much detail as necessary to understand it. 
+
+*Writing Sample*
+
+We work remotely on an open source project, so most of our communication with each other, our users, and external contributors happens through text. Send us some writing that you're proud of. This could be something like:
+* Block comment in source code
+* Blog post or other long-form article
+* Github or forum post (preferably discussing a technical topic)
+* Design document or specification
+* Whitepaper or academic publication
+
+*Analysis Sample*
+
+Zcash is a Bitcoin fork with added privacy features, so we sometimes debug edge cases in an underlying protocol we didn't design (though we also subject our new designs to intense review and analysis). The ability to pick up a complex existing system and figure out how it works or what might be causing a bug is a critical skill for us. In the analysis sample, tell us about a time you had to analyze a technical system whose abstractions failed you. This could be anything that required developing a deeper understanding, such as a memorable debugging session or a satisfying performance improvement.
+
+*Presentation Sample*
+It's important to be able to make our work legible both to our collaborators and to the public we serve. Along with writing well, oral presentation skills are an important vehicle for this kind of communication. If you've given a technical presentation that is publicly available, please provide a link to the slides and video (if available).
```

Comment:
Newline so that the fake heading doesn't run into the paragraph

```suggestion
*Presentation Sample*

It's important to be able to make our work legible both to our collaborators and to the public we serve. Along with writing well, oral presentation skills are an important vehicle for this kind of communication. If you've given a technical presentation that is publicly available, please provide a link to the slides and video (if available).
```